app/screens/interactive_dialog/dialog_introduction_text.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+
+import Markdown from 'app/components/markdown';
+
+import {getMarkdownTextStyles, getMarkdownBlockStyles} from 'app/utils/markdown';
+import {makeStyleSheetFromTheme} from 'app/utils/theme';
+
+export default class DialogIntroductionText extends PureComponent {
+    static propTypes = {
+        value: PropTypes.string.isRequired,
+        theme: PropTypes.object.isRequired,
+    };
+
+    render() {
+        const style = getStyleFromTheme(this.props.theme);
+        const blockStyles = getMarkdownBlockStyles(this.props.theme);
+        const textStyles = getMarkdownTextStyles(this.props.theme);
+
+        return (
+            <View style={style.introductionTextView}>
+                <Markdown
+                    baseTextStyle={style.introductionText}
+                    textStyles={textStyles}
+                    blockStyles={blockStyles}
+                    value={this.props.value}
+                    disableHashtags={true}
+                    disableAtMentions={true}
+                    disableChannelLink={true}
+                />
+            </View>
+        );
+    }
+}
+
+const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
+    return {
+        introductionTextView: {
+            marginHorizontal: 15,
```

Comment:
I'm almost sure this is 16, if its not we can leave it at 15 but we should file a follow up ticket to make everything 16 pt margin.

For now no changes needed, unless @esethna says otherwise

Reply:
I looked at the `DialogElement` component, which renders `TextSetting`, and noticed that most of the margins are set to 15, which I verified through the inspector (e.g. https://github.com/mattermost/mattermost-mobile/blob/master/app/components/widgets/settings/text_setting.js#L239).   I was debating whether I should fix the font size to 14 to match the `TextSetting` `title` style.  

I can change either if you want.

Reply:
we can leave it as is